task-2/src/components/Formik/AddMovie/index.js

```diff
@@ -0,0 +1,73 @@
+import React, { useState } from 'react';
+
+import { useDispatch } from 'react-redux';
+
+import { Formik, Form } from 'formik';
+import * as Yup from 'yup';
+
+import { submitMovie } from '../../../store/actionCreators';
+import { formikFieldOptions } from '../../../constants';
+
+import { FormikField } from '../FormikField';
+
+import './index.css';
+
+const movieValidationSchema = Yup.object().shape({
```

Comment:
Consider extracting validation into a separate file next to the target file (the one using it) or into "core" level

Reply:
fixed.